### ISSUE TITLE
Hierarchical lists don't work anymore

### ISSUE THREAD

**Issue by tethysgeco** [3liz/lizmap-web-client#1372]
### What is the bug?
Hierarchical lists don't work in locate by layer with QGIS 3.4 and QGIS server 3. It seems that the 2 separate vector layers work in indipendent way.
The same project works properly with QGIS and QGIS server 2.18.

### Environment
 - Lizmap version: 3.3.0
 - QGIS Server version: 3
 - QGIS Project version: 3.4
 - OS (Windows, Linux, MacOS, Android…): Debian
 - Browser (Firefox, Chrome…): Chrome
 - PostgreSQL

**Comment by mdouchin** [comment]
I think commit b4c1dac1f3daea760108fbe69b5df3c9941938ad solves this issue

**Comment by tethysgeco** [comment]
I've just tried but it doesn't work. 
I noticed also a strange behaviuor in the attribute table: the first time I select a record in the parent tabel no data in children are displayed. At the second click on the parent table it starts to work properly. 

The following image display the error at first click on parent tabel.
![error](https://user-images.githubusercontent.com/23027624/68130315-6ed28100-ff1b-11e9-970f-8f4f38f5b8e8.JPG)
